lib/tasks/auto_annotate_models.rake

```diff
@@ -0,0 +1,59 @@
+# NOTE: only doing this in development as some production environments (Heroku)
```

Comment:
Ohh you just added it 👌🏽 

I think you don't need to remember this any longer, and also keep the README as it was

Reply:
You are right. Thanks